contrib/test/ci/build/cri-o.yml

```diff
@@ -109,3 +109,5 @@
       dest: /etc/containers/registries.d/registry.access.redhat.com.yaml
     - src: test/default.yaml
       dest: /etc/containers/registries.d/default.yaml
+    - src: test/99-log-level.conf
```

Comment:
nit. It seems that we usually add a name to these types of configs (crun/log-level).

Reply:
```
- name: use crun
  copy:
    dest: /etc/crio/crio.conf.d/01-crun.conf
    content: |
      [crio.runtime]
      default_runtime = "crun"
      [crio.runtime.runtimes.crun]
      runtime_root = "/run/crun"
  when: "build_crun | default(False) | bool"

- name: use conmon-rs
  copy:
    dest: /etc/crio/crio.conf.d/99-conmonrs.conf
    content: |
      [crio.runtime.runtimes.runc]
      runtime_type = "pod"

      [crio.runtime.runtimes.crun]
      runtime_type = "pod"
      runtime_root = "/run/crun"
  when: "use_conmonrs | default(False) | bool"

- name: use evented-pleg
  copy:
    dest: /etc/crio/crio.conf.d/99-evented-pleg.conf
    content: |
      [crio.runtime]
      enable_pod_events = true
  when: "evented_pleg_fg | default(False) | bool"
```

Its not a big deal either way but do we want people to add configs in the "install-config" section or to add a section for adding new fields to logs?

Reply:
Using a dedicated section now :+1: